src/python/grpcio/grpc/__init__.py

```diff
@@ -299,6 +300,20 @@ def add_callback(self, callback):
     raise NotImplementedError()
 
 
+############################  gRPC Metadata  ##################################
+
+
+class Metadatum(collections.namedtuple('Metadatum', ('key', 'value'))):
```

Comment:
While it's true that we have a few concrete classes in the API they're all instances of our having been forced to have them by the shape of the underlying core API. That isn't the case here, so this should be an interface.

`HandlerCallDetails` would be a good example after which to model this class except for where I forgot the mandatory blank line in its doc string. :-P